### ISSUE TITLE
please make a video explaining how to use this on for example battlefront please on screen instruction can be very helpful.

### ISSUE THREAD

**Issue by gamedev44** [CallumCVM/FrostbiteGen#1]

**Comment by HexPerson** [comment]
No. The reposity is provided as-is and its purely code, if you can't get it to work then that could be one of the following problems:

1. You're incompetent
2. The code is over 10 years old

**Comment by CallumCVM** [comment]
No. The reposity is provided as-is and its purely code, if you can't get it to work then that could be one of the following problems:

You're incompetent
The code is over 10 years old
